src/engine/qcommon/common.cpp

```diff
@@ -912,25 +912,28 @@ void Com_Frame()
 		{
 			if ( com_minimized->integer && maxfpsMinimized.Get() > 0 )
 			{
-				minMsec = 1000 / maxfpsMinimized.Get();
+				minMsec = 1000 / std::min(maxfpsMinimized.Get(), 333);
 			}
 			else if ( com_unfocused->integer && maxfpsUnfocused.Get() > 0 )
 			{
-				minMsec = 1000 / maxfpsUnfocused.Get();
+				minMsec = 1000 / std::min(maxfpsUnfocused.Get(), 333);
 			}
 			else if ( maxfps.Get() > 0 )
 			{
-				minMsec = 1000 / maxfps.Get();
+				minMsec = 1000 / std::min(maxfps.Get(), 333);
 			}
 			else
 			{
-				minMsec = 1;
+				minMsec = 3;
 			}
 		}
 	}
 	else
 	{
-		minMsec = 1; // Bad things happen if this is 0
+		// Bad things happen if this is 0.
+		// At 1 or 2, the game still runs but exhibits various issues
+		// (such as the first-person weapon model flickering).
+		minMsec = 3;
```

Comment:
If I'm right timedemo is to benchmark things, I wonder if that can cap benchmarks?

Reply:
I'm not sure if timedemo influences the maximum effective FPS that can be reached (since the game will run faster than normal). Logically, I'd say yes, but I haven't tested this in any id Tech 3 game.

Reply:
I tested with a demo I had on my storage:

With 3:
```
7932 frames, 28.8s: 275.6 fps
7932 frames, 28.6s: 277.6 fps
```

With 1:
```
7932 frames, 25.9s: 305.7 fps
7932 frames, 25.9s: 306.1 fps
```

Note: I have not noticed weapon disappearing issue with timedemo.

Reply:
I also recorded a demo while weapon was flickering and another demo while weapon wasn't flickering (capped at 333fps), and both timedemoed demo looked fine with `minMsec 1`.